feedback/midcourse-feedback-feedback.md

```diff
@@ -0,0 +1,84 @@
+# Mid-Course Feedback Feedback
+
+
+## Jesse
+
+- Slow down!
+- Include more examples from past industry experiences
+- Find a balance between giving away an answer and challenging the student
+- Check in more often with students during in-class exercises
+
+## Erica
+
+- Prepare more thoroughly for lessons, including anticipating potential student questions
+- Communicate the material more clearly -- avoid mumbling, take control of timing, and be careful about calling on the supporting instructor for help
+- Avoid getting flustered/side-tracked while teaching, both in lectures and one-on-one. Make sure to follow up with students ASAP if we get interrupted while working on something.
+
+## Adam
+
+- Share more industry examples
+- Connect code / lessons to the real world more
+- Be more personable
+
+## Robin
+
+- Cold-call on students more so that there's less responsibility on them to raise their hands when they're feeling lost
+- Approach students more, instead of waiting for students to approach me
+- Share more industry examples
+
+## Andy
+
+- Use more politically correct examples
```

Comment:
"Politically correct" is kind of a loaded term, can be interpreted as insincere. Maybe something like "Use more appropriate humor"